lib/rubygems/installer.rb

```diff
@@ -698,24 +698,23 @@ def process_options # :nodoc:
 
   def check_that_user_bin_dir_is_in_path # :nodoc:
     user_bin_dir = @bin_dir || Gem.bindir(gem_home)
-    user_bin_dir = user_bin_dir.gsub(File::SEPARATOR, File::ALT_SEPARATOR) if
-      File::ALT_SEPARATOR
 
     path = ENV['PATH']
+
+    # Windows paths are case insensitive, but with either ALT_SEPARATOR
+    # or SEPARATOR
     if Gem.win_platform?
-      path = path.downcase
-      user_bin_dir = user_bin_dir.downcase
+      path = path.downcase.gsub(File::SEPARATOR, File::ALT_SEPARATOR)
```

Comment:
For single character replacement, I think `tr` is preferred (and faster).

Reply:
Once other issues resolved, I'll update.